tests/unit_tests/job_queue/test_lsf_driver.py

```diff
@@ -1,71 +1,117 @@
+import logging
 import os
+import re
 from argparse import ArgumentParser
+from datetime import datetime
 from pathlib import Path
 from textwrap import dedent
+from unittest.mock import patch
 
 import pytest
 
 from ert.__main__ import ert_parser
 from ert.cli import ENSEMBLE_EXPERIMENT_MODE
 from ert.cli.main import run_cli
+from ert.job_queue.driver import LSFDriver
+
+
+@pytest.fixture
+def mock_bsub(tmp_path):
+    script_path = tmp_path / "mock_bsub"
+    script_path.write_text(
+        "#!/usr/bin/env python3"
+        + dedent(
+            """
+                import sys
+                import time
+                import random
+                run_path = sys.argv[-1]
+                with open("job_paths", "a+", encoding="utf-8") as jobs_file:
```

Comment:
avoid "job" when you can.

Reply:
or, here we might refer to the LSF concept of a "Job", then it might be fine.

Reply:
Yes, it is taken from the LSF context.